### PR TITLE
Switch to n-layout

### PR DESCRIPTION
**Pull request by charypar**
Us n-layout instead of next-express layouts

/cc @i-like-robots

### CODE REVIEW

**Comment by charypar** [comment]
Don't merge yet, I have a suspicion it breaks the top nav somehow.

**Comment by charypar** [comment]
Yep, the "More" item in n-header-footer stopped toggling.

**Comment by charypar** [comment]
That's fixed now

File: client/main.scss [hunk 1]
```diff
@@ -21,6 +21,7 @@ $_next-ui-setup-first-pass: false;
 @import "colors";
 @import "type";
 
+@import "n-layout/main";
 @import "n-header-footer/main";
```
Comment (i-like-robots) [inline client/main.scss]: remove this

File: client/main.scss [hunk 2]
```diff
@@ -21,6 +21,7 @@ $_next-ui-setup-first-pass: false;
...
 @import "next-welcome/main";
```
Comment (i-like-robots) [inline client/main.scss]: and this

**Comment by i-like-robots** [comment]
:ship:

**Comment by tom-parker** [comment]
@charypar is anything stopping this getting merged?

**Pull request merged**